includes/admin/views/html-admin-dashboard-setup.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Admin View: Dashboard - Finish Setup
+ *
+ * @package WooCommerce\Admin
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+<div class="dashboard-widget-finish-setup">
+	<span class='progress-wrapper'>
+		<svg class="circle-progress" width="17" height="17" version="1.1" xmlns="http://www.w3.org/2000/svg">
+		  <circle r="6.5" cx="10" cy="10" fill="transparent" stroke-dasharray="40.859" stroke-dashoffset="0"></circle>
+		  <circle class="bar" r="6.5" cx="190" cy="10" fill="transparent" stroke-dasharray="40.859" stroke-dashoffset="<?php echo esc_attr( $circle_dashoffset ); ?>" transform='rotate(-90 100 100)'></circle>
+		</svg>
+		<span><?php echo esc_html_e( 'Step', 'woocommerce' ); ?> <?php echo esc_html( $completed_tasks_count ); ?> <?php echo esc_html_e( 'of', 'woocommerce' ); ?> <?php echo esc_html( $tasks_count ); ?></span>
+	</span>
+
+	<div class="description">
+		<div>
+			<?php echo esc_html_e( 'You\'re almost there! Once you complete store setup you can start receiving orders.', 'woocommerce' ); ?>
+			<div><a href='<?php echo esc_attr( $button_link ); ?>' class='button button-primary'><?php echo esc_html_e( 'Start selling', 'woocommerce' ); ?></a></div>
+		</div>
+		<img src="<?php echo esc_url( WC()->plugin_url() ); ?>/assets/images/dashboard-widget-setup.png"" />
```

Comment:
What is this image used for? Also it looks like there is a doubled quotation mark at the end of the `src` attribute.

Reply:
Thank you for catching it. I will remove the extra double quote. Interesting my IDE did not catch that..

The image is for the person on the right side.  

This guy :) 

![Screen Shot 2021-02-22 at 6 15 18 PM](https://user-images.githubusercontent.com/4723145/108794058-f04b8980-7539-11eb-8ec2-0f5cec3e1dcd.jpg)